rules/coding-style/src/Node/NameImporter.php

```diff
@@ -191,6 +192,13 @@ private function isNamespaceOrUseImportName(Name $name): bool
     private function isFunctionOrConstantImportWithSingleName(Name $name): bool
     {
         $parentNode = $name->getAttribute(AttributeKey::PARENT_NODE);
+
+        $fullName = $name->toCodeString();
```

Comment:
use `->toString()` instead to get cleaned up FQN, eg, when Name is a `FullyQualified`, after that, use `! Strings::contains($fullName, '\\')` to ensure this patch only applied to simple function.

```suggestion
        $fullName = $name->toString();
```